sci-libs/Shapely/Shapely-1.7.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
```

Comment:
```
running build_ext
skipping 'shapely/vectorized/_vectorized.c' Cython extension (up-to-date)
building 'shapely.speedups._speedups' extension
building 'shapely.vectorized._vectorized' extension
creating /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9
creating /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely
creating /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely
creating /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely/speedups
creating /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely/vectorized
x86_64-pc-linux-gnu-gcc -march=native -O2 -pipe -frecord-gcc-switches -fPIC -I/usr/include -I/usr/lib/python3.9/site-packages/numpy/core/include -I/usr/include/python3.9 -c shapely/vectorized/_vectorized.c -o /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely/vectorized/_vectorized.o
x86_64-pc-linux-gnu-gcc -march=native -O2 -pipe -frecord-gcc-switches -fPIC -I/usr/include -I/usr/lib/python3.9/site-packages/numpy/core/include -I/usr/include/python3.9 -c shapely/speedups/_speedups.c -o /var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0-python3_9/temp.linux-x86_64-3.9/shapely/speedups/_speedups.o
In file included from /usr/lib/python3.9/site-packages/numpy/core/include/numpy/ndarraytypes.h:1822,
                 from /usr/lib/python3.9/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /usr/lib/python3.9/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from shapely/vectorized/_vectorized.c:242:
/usr/lib/python3.9/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
shapely/speedups/_speedups.c: In function '__Pyx_InitCachedConstants':
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8818:62: note: in expansion of macro 'CO_OPTIMIZED'
 8818 |   __pyx_codeobj__4 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__3, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 161, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__4)) __PYX_ERR(0, 161, __pyx_L1_error)
      |                                                              ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8818:243: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8818 |   __pyx_codeobj__4 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__3, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 161, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__4)) __PYX_ERR(0, 161, __pyx_L1_error)
      |                                                                                                                                                                                                                                                   ^~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                   |
      |                                                                                                                                                                                                                                                   PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8818:263: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8818 |   __pyx_codeobj__4 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__3, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 161, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__4)) __PYX_ERR(0, 161, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                       ^~~
      |                                                                                                                                                                                                                                                                       |
      |                                                                                                                                                                                                                                                                       int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8818:33: note: in expansion of macro '__Pyx_PyCode_New'
 8818 |   __pyx_codeobj__4 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__3, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 161, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__4)) __PYX_ERR(0, 161, __pyx_L1_error)
      |                                 ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8863:62: note: in expansion of macro 'CO_OPTIMIZED'
 8863 |   __pyx_codeobj__9 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__8, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 332, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__9)) __PYX_ERR(0, 332, __pyx_L1_error)
      |                                                              ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8863:243: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8863 |   __pyx_codeobj__9 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__8, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 332, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__9)) __PYX_ERR(0, 332, __pyx_L1_error)
      |                                                                                                                                                                                                                                                   ^~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                   |
      |                                                                                                                                                                                                                                                   PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8863:263: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8863 |   __pyx_codeobj__9 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__8, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 332, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__9)) __PYX_ERR(0, 332, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                       ^~~
      |                                                                                                                                                                                                                                                                       |
      |                                                                                                                                                                                                                                                                       int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8863:33: note: in expansion of macro '__Pyx_PyCode_New'
 8863 |   __pyx_codeobj__9 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__8, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coords_2, 332, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__9)) __PYX_ERR(0, 332, __pyx_L1_error)
      |                                 ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8908:63: note: in expansion of macro 'CO_OPTIMIZED'
 8908 |   __pyx_codeobj__15 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__14, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_destroy, 45, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__15)) __PYX_ERR(0, 45, __pyx_L1_error)
      |                                                               ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8908:245: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8908 |   __pyx_codeobj__15 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__14, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_destroy, 45, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__15)) __PYX_ERR(0, 45, __pyx_L1_error)
      |                                                                                                                                                                                                                                                     ^~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                     |
      |                                                                                                                                                                                                                                                     PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8908:264: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8908 |   __pyx_codeobj__15 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__14, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_destroy, 45, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__15)) __PYX_ERR(0, 45, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                        ^~
      |                                                                                                                                                                                                                                                                        |
      |                                                                                                                                                                                                                                                                        int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8908:34: note: in expansion of macro '__Pyx_PyCode_New'
 8908 |   __pyx_codeobj__15 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__14, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_destroy, 45, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__15)) __PYX_ERR(0, 45, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8920:63: note: in expansion of macro 'CO_OPTIMIZED'
 8920 |   __pyx_codeobj__17 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__16, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_required, 49, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__17)) __PYX_ERR(0, 49, __pyx_L1_error)
      |                                                               ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8920:245: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8920 |   __pyx_codeobj__17 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__16, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_required, 49, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__17)) __PYX_ERR(0, 49, __pyx_L1_error)
      |                                                                                                                                                                                                                                                     ^~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                     |
      |                                                                                                                                                                                                                                                     PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8920:265: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8920 |   __pyx_codeobj__17 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__16, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_required, 49, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__17)) __PYX_ERR(0, 49, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                         ^~
      |                                                                                                                                                                                                                                                                         |
      |                                                                                                                                                                                                                                                                         int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8920:34: note: in expansion of macro '__Pyx_PyCode_New'
 8920 |   __pyx_codeobj__17 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__16, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_required, 49, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__17)) __PYX_ERR(0, 49, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8932:64: note: in expansion of macro 'CO_OPTIMIZED'
 8932 |   __pyx_codeobj__19 = (PyObject*)__Pyx_PyCode_New(3, 0, 19, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__18, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linestring_from_py, 59, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__19)) __PYX_ERR(0, 59, __pyx_L1_error)
      |                                                                ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8932:246: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8932 |   __pyx_codeobj__19 = (PyObject*)__Pyx_PyCode_New(3, 0, 19, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__18, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linestring_from_py, 59, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__19)) __PYX_ERR(0, 59, __pyx_L1_error)
      |                                                                                                                                                                                                                                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                      |
      |                                                                                                                                                                                                                                                      PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8932:281: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8932 |   __pyx_codeobj__19 = (PyObject*)__Pyx_PyCode_New(3, 0, 19, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__18, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linestring_from_py, 59, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__19)) __PYX_ERR(0, 59, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                                         ^~
      |                                                                                                                                                                                                                                                                                         |
      |                                                                                                                                                                                                                                                                                         int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8932:34: note: in expansion of macro '__Pyx_PyCode_New'
 8932 |   __pyx_codeobj__19 = (PyObject*)__Pyx_PyCode_New(3, 0, 19, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__18, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linestring_from_py, 59, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__19)) __PYX_ERR(0, 59, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8944:64: note: in expansion of macro 'CO_OPTIMIZED'
 8944 |   __pyx_codeobj__21 = (PyObject*)__Pyx_PyCode_New(3, 0, 20, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__20, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linearring_from_py, 211, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__21)) __PYX_ERR(0, 211, __pyx_L1_error)
      |                                                                ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8944:246: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8944 |   __pyx_codeobj__21 = (PyObject*)__Pyx_PyCode_New(3, 0, 20, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__20, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linearring_from_py, 211, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__21)) __PYX_ERR(0, 211, __pyx_L1_error)
      |                                                                                                                                                                                                                                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                      |
      |                                                                                                                                                                                                                                                      PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8944:281: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8944 |   __pyx_codeobj__21 = (PyObject*)__Pyx_PyCode_New(3, 0, 20, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__20, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linearring_from_py, 211, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__21)) __PYX_ERR(0, 211, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                                         ^~~
      |                                                                                                                                                                                                                                                                                         |
      |                                                                                                                                                                                                                                                                                         int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8944:34: note: in expansion of macro '__Pyx_PyCode_New'
 8944 |   __pyx_codeobj__21 = (PyObject*)__Pyx_PyCode_New(3, 0, 20, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__20, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_geos_linearring_from_py, 211, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__21)) __PYX_ERR(0, 211, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8956:64: note: in expansion of macro 'CO_OPTIMIZED'
 8956 |   __pyx_codeobj__23 = (PyObject*)__Pyx_PyCode_New(1, 0, 10, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__22, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_ctypes, 405, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__23)) __PYX_ERR(0, 405, __pyx_L1_error)
      |                                                                ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8956:246: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8956 |   __pyx_codeobj__23 = (PyObject*)__Pyx_PyCode_New(1, 0, 10, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__22, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_ctypes, 405, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__23)) __PYX_ERR(0, 405, __pyx_L1_error)
      |                                                                                                                                                                                                                                                      ^~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                      |
      |                                                                                                                                                                                                                                                      PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8956:273: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8956 |   __pyx_codeobj__23 = (PyObject*)__Pyx_PyCode_New(1, 0, 10, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__22, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_ctypes, 405, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__23)) __PYX_ERR(0, 405, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                                 ^~~
      |                                                                                                                                                                                                                                                                                 |
      |                                                                                                                                                                                                                                                                                 int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8956:34: note: in expansion of macro '__Pyx_PyCode_New'
 8956 |   __pyx_codeobj__23 = (PyObject*)__Pyx_PyCode_New(1, 0, 10, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__22, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_ctypes, 405, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__23)) __PYX_ERR(0, 405, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
/usr/include/python3.9/cpython/code.h:67:25: warning: passing argument 6 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
   67 | #define CO_OPTIMIZED    0x0001
      |                         ^
      |                         |
      |                         int
shapely/speedups/_speedups.c:318:37: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                     ^
shapely/speedups/_speedups.c:8968:63: note: in expansion of macro 'CO_OPTIMIZED'
 8968 |   __pyx_codeobj__11 = (PyObject*)__Pyx_PyCode_New(1, 0, 8, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__24, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_iter, 430, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__11)) __PYX_ERR(0, 430, __pyx_L1_error)
      |                                                               ^~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:118:34: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  118 |         int, int, int, int, int, PyObject *, PyObject *,
      |                                  ^~~~~~~~~~
shapely/speedups/_speedups.c:8968:245: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
 8968 |   __pyx_codeobj__11 = (PyObject*)__Pyx_PyCode_New(1, 0, 8, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__24, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_iter, 430, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__11)) __PYX_ERR(0, 430, __pyx_L1_error)
      |                                                                                                                                                                                                                                                     ^~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                                                                     |
      |                                                                                                                                                                                                                                                     PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:8968:270: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
 8968 |   __pyx_codeobj__11 = (PyObject*)__Pyx_PyCode_New(1, 0, 8, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__24, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_iter, 430, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__11)) __PYX_ERR(0, 430, __pyx_L1_error)
      |                                                                                                                                                                                                                                                                              ^~~
      |                                                                                                                                                                                                                                                                              |
      |                                                                                                                                                                                                                                                                              int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:8968:34: note: in expansion of macro '__Pyx_PyCode_New'
 8968 |   __pyx_codeobj__11 = (PyObject*)__Pyx_PyCode_New(1, 0, 8, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__24, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_shapely_speedups__speedups_pyx, __pyx_n_s_coordseq_iter, 430, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__11)) __PYX_ERR(0, 430, __pyx_L1_error)
      |                                  ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
shapely/vectorized/_vectorized.c: In function 'PyInit__vectorized':
shapely/vectorized/_vectorized.c:18675:25: error: 'PyTypeObject' {aka 'struct _typeobject'} has no member named 'tp_print'
18675 |   __pyx_type___pyx_array.tp_print = 0;
      |                         ^
shapely/vectorized/_vectorized.c:18678:31: error: 'PyTypeObject' {aka 'struct _typeobject'} has no member named 'tp_print'
18678 |   __pyx_type___pyx_MemviewEnum.tp_print = 0;
      |                               ^
shapely/vectorized/_vectorized.c:18689:30: error: 'PyTypeObject' {aka 'struct _typeobject'} has no member named 'tp_print'
18689 |   __pyx_type___pyx_memoryview.tp_print = 0;
      |                              ^
shapely/vectorized/_vectorized.c:18698:35: error: 'PyTypeObject' {aka 'struct _typeobject'} has no member named 'tp_print'
18698 |   __pyx_type___pyx_memoryviewslice.tp_print = 0;
      |                                   ^
shapely/speedups/_speedups.c: In function '__Pyx_CreateCodeObjectForTraceback':
shapely/speedups/_speedups.c:12017:9: warning: passing argument 14 of 'PyCode_New' makes integer from pointer without a cast [-Wint-conversion]
12017 |         py_funcname,  /*PyObject *name,*/
      |         ^~~~~~~~~~~
      |         |
      |         PyObject * {aka struct _object *}
shapely/speedups/_speedups.c:318:69: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                     ^~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:33: note: expected 'int' but argument is of type 'PyObject *' {aka 'struct _object *'}
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                 ^~~
shapely/speedups/_speedups.c:12018:9: warning: passing argument 15 of 'PyCode_New' makes pointer from integer without a cast [-Wint-conversion]
12018 |         py_line,
      |         ^~~~~~~
      |         |
      |         int
shapely/speedups/_speedups.c:318:75: note: in definition of macro '__Pyx_PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |                                                                           ^~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:120:38: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'int'
  120 |         PyObject *, PyObject *, int, PyObject *);
      |                                      ^~~~~~~~~~
shapely/speedups/_speedups.c:318:11: error: too many arguments to function 'PyCode_New'
  318 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
      |           ^~~~~~~~~~
shapely/speedups/_speedups.c:12004:15: note: in expansion of macro '__Pyx_PyCode_New'
12004 |     py_code = __Pyx_PyCode_New(
      |               ^~~~~~~~~~~~~~~~
In file included from /usr/include/python3.9/code.h:13,
                 from /usr/include/python3.9/Python.h:117,
                 from shapely/speedups/_speedups.c:4:
/usr/include/python3.9/cpython/code.h:117:28: note: declared here
  117 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
      |                            ^~~~~~~~~~
shapely/vectorized/_vectorized.c: In function '__Pyx_ExceptionSave':
shapely/vectorized/_vectorized.c:20415:21: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20415 |     *type = tstate->exc_type;
      |                     ^~~~~~~~
      |                     curexc_type
shapely/vectorized/_vectorized.c:20416:22: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20416 |     *value = tstate->exc_value;
      |                      ^~~~~~~~~
      |                      curexc_value
shapely/vectorized/_vectorized.c:20417:19: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20417 |     *tb = tstate->exc_traceback;
      |                   ^~~~~~~~~~~~~
      |                   curexc_traceback
shapely/vectorized/_vectorized.c: In function '__Pyx_ExceptionReset':
shapely/vectorized/_vectorized.c:20429:24: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20429 |     tmp_type = tstate->exc_type;
      |                        ^~~~~~~~
      |                        curexc_type
shapely/vectorized/_vectorized.c:20430:25: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20430 |     tmp_value = tstate->exc_value;
      |                         ^~~~~~~~~
      |                         curexc_value
shapely/vectorized/_vectorized.c:20431:22: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20431 |     tmp_tb = tstate->exc_traceback;
      |                      ^~~~~~~~~~~~~
      |                      curexc_traceback
shapely/vectorized/_vectorized.c:20432:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20432 |     tstate->exc_type = type;
      |             ^~~~~~~~
      |             curexc_type
shapely/vectorized/_vectorized.c:20433:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20433 |     tstate->exc_value = value;
      |             ^~~~~~~~~
      |             curexc_value
shapely/vectorized/_vectorized.c:20434:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20434 |     tstate->exc_traceback = tb;
      |             ^~~~~~~~~~~~~
      |             curexc_traceback
shapely/vectorized/_vectorized.c: In function '__Pyx_GetException':
shapely/vectorized/_vectorized.c:20477:24: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20477 |     tmp_type = tstate->exc_type;
      |                        ^~~~~~~~
      |                        curexc_type
shapely/vectorized/_vectorized.c:20478:25: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20478 |     tmp_value = tstate->exc_value;
      |                         ^~~~~~~~~
      |                         curexc_value
shapely/vectorized/_vectorized.c:20479:22: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20479 |     tmp_tb = tstate->exc_traceback;
      |                      ^~~~~~~~~~~~~
      |                      curexc_traceback
shapely/vectorized/_vectorized.c:20480:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20480 |     tstate->exc_type = local_type;
      |             ^~~~~~~~
      |             curexc_type
shapely/vectorized/_vectorized.c:20481:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20481 |     tstate->exc_value = local_value;
      |             ^~~~~~~~~
      |             curexc_value
shapely/vectorized/_vectorized.c:20482:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20482 |     tstate->exc_traceback = local_tb;
      |             ^~~~~~~~~~~~~
      |             curexc_traceback
shapely/vectorized/_vectorized.c: In function '__Pyx_ExceptionSwap':
shapely/vectorized/_vectorized.c:20504:24: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20504 |     tmp_type = tstate->exc_type;
      |                        ^~~~~~~~
      |                        curexc_type
shapely/vectorized/_vectorized.c:20505:25: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20505 |     tmp_value = tstate->exc_value;
      |                         ^~~~~~~~~
      |                         curexc_value
shapely/vectorized/_vectorized.c:20506:22: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20506 |     tmp_tb = tstate->exc_traceback;
      |                      ^~~~~~~~~~~~~
      |                      curexc_traceback
shapely/vectorized/_vectorized.c:20507:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_type'; did you mean 'curexc_type'?
20507 |     tstate->exc_type = *type;
      |             ^~~~~~~~
      |             curexc_type
shapely/vectorized/_vectorized.c:20508:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_value'; did you mean 'curexc_value'?
20508 |     tstate->exc_value = *value;
      |             ^~~~~~~~~
      |             curexc_value
shapely/vectorized/_vectorized.c:20509:13: error: 'PyThreadState' {aka 'struct _ts'} has no member named 'exc_traceback'; did you mean 'curexc_traceback'?
20509 |     tstate->exc_traceback = *tb;
      |             ^~~~~~~~~~~~~
      |             curexc_traceback
WARNING:/var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0/setup.py:command '/usr/bin/x86_64-pc-linux-gnu-gcc' failed with exit code 1
WARNING:/var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0/setup.py:The C extension could not be compiled, speedups are not enabled.
WARNING:/var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0/setup.py:Failure information, if any, is above.
WARNING:/var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0/setup.py:I'm retrying the build without the C extension now.
running build
running build_py
running egg_info
writing Shapely.egg-info/PKG-INFO
writing dependency_links to Shapely.egg-info/dependency_links.txt
writing requirements to Shapely.egg-info/requires.txt
writing top-level names to Shapely.egg-info/top_level.txt
reading manifest file 'Shapely.egg-info/SOURCES.txt'
writing manifest file 'Shapely.egg-info/SOURCES.txt'
warning: build_py: byte-compiling is disabled, skipping.

WARNING:/var/tmp/portage/sci-libs/Shapely-1.7.0/work/Shapely-1.7.0/setup.py:The C extension could not be compiled, speedups are not enabled.
 * Using python3.9 in global scope
 * python3_9: running distutils-r1_run_phase python_compile_all
 * ERROR: sci-libs/Shapely-1.7.0::testworld failed (compile phase):
 *   docs/conf.py not found, distutils_enable_sphinx call wrong
```

It works all the way up to python-3.9. Any idea what could be wrong with 3.9? Is some dep not supported after all?

Reply:
@juippis sorry i've no idea. But i see the same warning with python 3.7 but it succeeds. Could it be related to the fact that python 3.9 treats some warnings now as errors? If so I would remove python 3.9 support from this ebuild.

Reply:
Well, in my eyes yes. If it can't be compiled with python-3.9 enabled, it's broken... Ideally I guess you'd want to report this to upstream for them to fix it.

Reply:
I've removed python 3.9 support and added an issue upstream [#944](https://github.com/Toblerity/Shapely/issues/944)